provider-ui/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "eslint .",
     "next": "next",
     "build": "next build",
-    "start": "next start",
+    "start": "NODE_ENV=production nodemon ui_dev_server.js",
```

Comment:
This might not be the best way to do it.

1. NODE_ENV is the place for Dockerfile/Docker-compose, it shouldn't really be here.
2. We shouldn't be using nodemon in production, it is a development only tool
    * We don't want the app to keep running after an exception occurs, nodemon will keep the server running even if it is intended to crash.
    * There are better ways to do it (something like pm2)
    * nodemon is a dev dependency and won't be installed on the server
3. Having said that, the `ui_dev_server.js` file is specifically written for use during development. Are you sure that the correct host and port for communicating with API in production server is `http://localhost:9081`? ([line28](https://github.com/layer5io/meshery/blob/94a363e1cbb736cb3c4411566f6335c0729e26a6/provider-ui/ui_dev_server.js#L28))

4. Just to make sure, am I right when I say we are using the next.js server, and NOT the static build of the UI? @leecalcote

Reply:
I think you really need to get acquainted with next.js and ssr. Hope this will help you https://nextjs.org/docs/advanced-features/custom-server

Reply:
Yes I looked through that page @kushthedude 
I was just making sure we are actually making use of SSR

Reply:
Now that I know we are using SSR, maybe the best way to do it will be `"start": "node ui_dev_server.js" `

Reply:
We are using next export (https://github.com/layer5io/meshery/blob/master/Dockerfile#L10) . We are using npm start only for development purposes. Then again NODE_ENV should not be production in such case.

Reply:
Oh, thanks @nveenjain. 
Since we are not using SSR (yet), and we are using a separate dev script for development, there is no need for a start script for now.
So probably we should best have it as `"start": "node ui_dev_server.js"`